docs/subnets/deploying-subnets-on-prod.md

```diff
@@ -0,0 +1,191 @@
+
+Of course, Subnet-CLI supports the usage of a Ledger HW wallet. Take advantage of that because losing control of the managing would be catastrophic. General instruction on how to use a Ledger device with Avalanche can be found [here](https://support.avax.network/en/articles/6150237-how-to-use-a-ledger-nano-s-or-nano-x-with-avalanche).
+
+### Genesis File
+
+The structure that defines the most important parameters in a Subnet is found in the genesis file, which is a `json` formatted, human-readable file. Describing the contents and the options available in the genesis file is beyond the scope of this document, and if you're ready to deploy your Subnet to production you probably have it mapped out already. 
+
+If you want to review, we have a description of the genesis file in our document on [customizing EVM Subnets](customize-a-subnet.md).
+
+### Subnet-CLI Wizard
+
+Creating a Subnet is a multistep process. You need to:
+* create a `WMID`
+* create the Subnet
+* create the blockchain in the Subnet
+* add validators to Subnet
+
+Fortunately, to minimize potential errors and streamline the process Subnet-CLi has a Wizard command that does most of the work for you snd guides you through the process.
+
+Before running the wizard you will need to [create the `WMID`](subnet-cli.md#subnet-cli-create-vmid), have the validator NodeIDs ready, as well as the genesis `json` file.
+
+An example of wizard command line could look like:
+
+```bash
+subnet-cli wizard \
+--ledger \
+--node-ids=NodeID-741aqvs6R4iuHDyd1qT1NrFTmsgu78dc4,NodeID-K7Y79oAmBntAcdkyY1CLxCim8QuqcZbBp,NodeID-C3EY6u4v7DDi6YEbYf1wmXdvkEFXYuXNW,NodeID-AiLGeqQfh9gZY3Y8wLMD15tuJtsJHq5Qi \
+--vm-genesis-path=prod-genesis.json \
+--vm-id=tGas3T58KzdjLHhBDMnH2TvrddhqTji5iZAMZ3RXs2NLpSnhH \
+--chain-name=prodSubnet
+```
+
+Please refer to the Subnet-CLI docs for [detailed instructions](subnet-cli.md#subnet-cli-wizard).
+
+If the wizard completes correctly your Subnet will now be created, deployed and available to see in [the Subnet explorer](https://subnets.avax.network/subnets). Congrats!
+
+You can also use Subnet-CLI to manage your Subnet after the initial deploy, to add new validators, renew ones that expired and so on. Familiarize yourself with Subnet-CLI, you be using it often!
+
+## Validator Configuration
+
+Running nodes as Subnet validators warrants some additional considerations, above those when running a regular node or a Primary Network-only validator.
+
+### Subnet Whitelisting
+
+For a node to join a subnet, there are two prerequisites:
+* Primary Network validation
```

Comment:
[MD032/blanks-around-lists: Lists should be surrounded by blank lines](https://github.com/DavidAnson/markdownlint/blob/v0.25.1/doc/Rules.md#md032---lists-should-be-surrounded-by-blank-lines)